os/board/rtl8730e/src/rtl8730e_mipi_lcdc.c

```diff
@@ -207,6 +223,13 @@ static void rtl8730e_control_backlight(uint8_t level)
 #if defined(CONFIG_LCD_ST7785) || defined(CONFIG_LCD_ST7701SN)
 	pwmout_write(&g_rtl8730e_config_dev_s.pwm_led, 1.0-pwm_level);
```

Comment:
Hi, currently, the resume flow is handled in the wakeup hook. However, since this flow duplicates rtk_mipi_resume and is controlled by the application layer through power_on and backlight, it would be better to remove the following registration from amebasmart_mipi_dsi_host_initialize: pmu_register_sleep_callback(PMU_MIPI_DEVICE, (PSM_HOOK_FUN)rtk_mipi_suspend, NULL, (PSM_HOOK_FUN)rtk_mipi_resume, NULL);